timelock-impl/src/test/java/com/palantir/atlasdb/timelock/adjudicate/FeedbackAnalysisTest.java

```diff
@@ -0,0 +1,291 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.timelock.adjudicate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.After;
+import org.junit.Test;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import com.google.common.collect.ImmutableList;
+import com.palantir.timelock.feedback.ConjureTimeLockClientFeedback;
+import com.palantir.timelock.feedback.EndpointStatistics;
+
+public class FeedbackAnalysisTest {
+    private static final FakeTicker FAKE_TICKER = new FakeTicker();
+    private static final String CLIENT = "client_1";
+    private static final String CLIENT_2 = "client_2";
+    private static final String CLIENT_3 = "client_3";
+
+
+    private static TimeLockClientFeedbackSink timeLockClientFeedbackSink = TimeLockClientFeedbackSink
+            .create(Caffeine.newBuilder()
+                    .expireAfterWrite(Constants.HEALTH_FEEDBACK_REPORT_EXPIRATION_MINUTES, TimeUnit.MINUTES)
+                    .ticker(FAKE_TICKER)
+                    .build());
+
+    @After
+    public void resetCache() {
+        FAKE_TICKER.advance(Constants.HEALTH_FEEDBACK_REPORT_EXPIRATION_MINUTES, TimeUnit.MINUTES);
```

Comment:
I think the feedback sink should be non-static, in which case a new instance is created for each of the tests and therefore you wouldn't need to worry about this.